src/views/workflow-page/workflow-page-tabs/workflow-page-tabs.tsx

```diff
@@ -22,7 +25,14 @@ export default function WorkflowPageTabs() {
       setSelectedTab={(newTab) => {
         router.push(encodeURIComponent(newTab.toString()));
       }}
-      endEnhancer={<WorkflowPageCliCommandsButton />}
+      endEnhancer={
+        <styled.EndButtonsContainer>
+          <WorkflowPageCliCommandsButton />
+          <ErrorBoundary fallbackRender={() => null}>
```

Comment:
Not sure if we need this error boundary, since the entire page will fail to load if describeWorkflow fails